src/components/stars/starsHistory/starsHistory.tsx

```diff
@@ -83,8 +83,7 @@ const StarsHistory = ({ userRole, updateStar, platformToShow }: Props) => {
           || assigneeFilter.includes(star.assignee))
         && (computerFilter.length === 0
           || computerFilter.includes(star.computer))
-        && (resourceFilter.length === 0 || resourceFilter
-          .some((element) => star.resources.includes(element)))
+        && (phaseFilter.length === 0 || phaseFilter.includes(star.phase))
```

Comment:
[My thoughts right now](https://www.youtube.com/watch?v=BzUWKSYjpgQ)

Please split these conditions into smaller ones!

Reply:
:(

Reply:
Not necessary now, this PR can be merged without it, but please be careful, it could explode in the future

Reply:
Fixed although I'm a little hurt

Reply:
You should watch "full metal jacket" anyway!!!!!